### PR TITLE
add etcd icon

### PR DESCRIPTION
**Pull request by kerthcet**
### Checklist
  - [✔️ ] I updated the JSON data in `_data/simple-icons.json`
  - [✔️] I optimized the icon with SVGO or SVGOMG
  - [✔️ ] The SVG `viewbox` is `0 0 24 24`

### Description

homepage: https://etcd.io/
github star: 36.7k

I took svg from https://etcd.io/
I used https://jakearchibald.github.io/svgomg/ to optimize

I met a problem, I tried to follow the contributing.md to `Annotate the Icon`, but I have no idea how to do it. Can someone provide some help?

### CODE REVIEW

**Comment by kerthcet** [comment]
need some help😆

**Comment by dirien** [comment]
HI @kerthcet,

i think you have a problem with the viewbox. it should be 0 0 24 24 like my one ->
image
And you need to add the title tag too. 

What i do in the code is ->
```bash
npm install
npm run svgo -- icons/etcd.svg
npm run lint
```
That should give you the best feedback before the PR.

See for reference -> https://github.com/simple-icons/simple-icons/pull/6146/files

And i would add a pereview of your icon and color too! -> https://petershaggynoble.github.io/SI-Sandbox/preview/

**Comment by kerthcet** [comment]
> HI @kerthcet,
> 
> i think you have a problem with the viewbox. it should be 0 0 24 24 like my one ->
> image
> And you need to add the title tag too.
> 
> What i do in the code is ->
> 
> ```shell
> npm install
> npm run svgo -- icons/etcd.svg
> npm run lint
> ```
> 
> That should give you the best feedback before the PR.
> 
> See for reference -> https://github.com/simple-icons/simple-icons/pull/6146/files
> 
> And i would add a pereview of your icon and color too! -> https://petershaggynoble.github.io/SI-Sandbox/preview/

thansk for your help, l'll have a try.

**Comment by kerthcet** [comment]
> HI @kerthcet,
> 
> i think you have a problem with the viewbox. it should be 0 0 24 24 like my one ->
> image
> And you need to add the title tag too.
> 
> What i do in the code is ->
> 
> ```shell
> npm install
> npm run svgo -- icons/etcd.svg
> npm run lint
> ```
> 
> That should give you the best feedback before the PR.
> 
> See for reference -> https://github.com/simple-icons/simple-icons/pull/6146/files
> 
> And i would add a pereview of your icon and color too! -> https://petershaggynoble.github.io/SI-Sandbox/preview/

hey, @dirien if the `title` tag added manually?

**Comment by kerthcet** [comment]
Screen Shot 2021-07-25 at 12 44 47 AM

hey, @dirien Would you help to review my commit.🥷

**Comment by dirien** [comment]
Hi @kerthcet , sorry i can't as i am not a core contributor of this project. 😞

**Comment by kerthcet** [comment]
> Hi @kerthcet , sorry i can't as i am not a core contributor of this project. 😞

thanks anyway. Help me a lot.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2757,6 +2757,11 @@
             "hex": "E7352C",
             "source": "https://www.espressif.com/"
         },
+        {
+            "title": "Etcd",
+            "hex": "419EDA",
```
Comment (ericcornelissen) [inline _data/simple-icons.json:2762]: It seems you took this color from the SVG, is that correct? While the choice seems sensible I wondered if you looked around for any indication of the brand color of Etcd?

File: icons/etcd.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...1.986-.479 18.449 19.096 0 0 0 .267-3.638 18.185 18.823 0 0 0-2.274-2.79 4.52 4.678 0 0 1 1.603-1.777l.294-.189-.23-.268A11.977 12.398 0 0 0 16.808.143L16.488 0l-.08.347a4.506 4.665 0 0 1-1.134 2.131 18.19 18.829 0 0 0-3.277-1.397 18.127 18.763 0 0 0-3.273 1.395A4.495 4.653 0 0 1 7.594.35L7.512.001l-.317.143a12.092 12.516 0 0 0-4.182 3.157l-.23.268.294.189a4.512 4.67 0 0 1 1.6 1.767 18.323 18.966 0 0 0-2.27 2.78 18.436 19.083 0 0 0 .26 3.662 4.505 4.663 0 0 1-1.976.474 4.224 4.372 0 0 1-.346-.013L0 12.4l.032.356a11.904 12.323 0 0 0 1.61 5.097l.174.309.263-.232a4.502 4.66 0 0 1 2.131-1.033 18.29 18.932 0 0 0 1.824 3.076 18.474 19.122 0 0 0 3.457.88 4.493 4.65 0 0 1-.287 2.442l-.132.331.338.077A12.049 12.472 0 0 0 12 24l2.59-.297.339-.077-.131-.331a4.487 4.645 0 0 1-.287-2.444 18.477 19.125 0 0 0 3.443-.878 18.209 18.848 0 0 0 1.826-3.078 4.51 4.67 0 0 1 2.143 1.034l.263.23.175-.306a11.877 12.294 0 0 0 1.607-5.097L24 12.4zm-7.958 4.356a13.982 14.473 0 0 1-7.417 0 14.19 14.688 0 0 1-2.256-7.258 14.119 14.614 0 0 1 2.687-2.649 14.334 14.836 0 0 1 3.28-1.846 14.378 14.883 0 0 1 3.27 1.841 14.227 14.726 0 0 1 2.7 2.667 14.294 14.795 0 0 1-.675 3.78 14.366 14.87 0 0 1-1.59 3.465z"/></svg>
```
Comment (ericcornelissen) [inline icons/etcd.svg:1]: I assume you used the SVG in the top left of the website as the source, correct?

If so, it seems something didn't go quite right, see my comparison below. It looks like you resized the SVG to exactly 24x24 pixels. However, the icon isn't a perfect square so as a result the SVG is now distorted. Could you try this again but this time preserving the aspect ratio of the original?

image

File: _data/simple-icons.json [hunk 3]
```diff
@@ -2757,6 +2757,11 @@
             "hex": "E7352C",
             "source": "https://www.espressif.com/"
         },
+        {
+            "title": "Etcd",
```
Comment (ericcornelissen) [inline _data/simple-icons.json:2761]: I feel like their preferred way of writing the name is "etcd", not "Etcd". Would you agree? If so, could you update it here and in the SVG? If not, could you share your reasoning?

**Review by ericcornelissen — CHANGES_REQUESTED** [review]
We're sorry for the delay on this review, please understand that this is a project ran entirely by volunteers - as a result sometimes reviews can take a bit. I hope you understand. As for the contents of this Pull Request, see my comments below :slightly_smiling_face: 

Thank you very much for jumping in to help out @dirien! :100:

**Comment by kerthcet** [comment]
> We're sorry for the delay on this review, please understand that this is a project ran entirely by volunteers - as a result sometimes reviews can take a bit. I hope you understand. As for the contents of this Pull Request, see my comments below 🙂
> 
> And, thank you very much for jumping in to help out @dirien! 💯

I understood, and i'm glad to join the volunteer work if I met the requirements one day.

File: _data/simple-icons.json (hunk 3 above)
Comment (kerthcet) [reply _data/simple-icons.json:2761]: done :)

File: _data/simple-icons.json (hunk 1 above)
Comment (kerthcet) [reply _data/simple-icons.json:2762]: I pick the brand color of etcd from `https://cncf-branding.netlify.app/projects/etcd/`, I think it's convinced.🧐 and i'll change the source address to this website as the same.

File: icons/etcd.svg (hunk 2 above)
Comment (kerthcet) [reply icons/etcd.svg]: corrected already

**Comment by kerthcet** [comment]
thanks for your patience. @ericcornelissen

File: _data/simple-icons.json (hunk 1 above)
Comment (ericcornelissen) [reply _data/simple-icons.json:2762]: That is perfect!

That wasn't the source when I reviewed this right :sweat_smile: If it was, I just glossed right over the color...

File: icons/etcd.svg (hunk 2 above)
Comment (ericcornelissen) [reply icons/etcd.svg]: Perfect, nice work! :100:

**Review by ericcornelissen — APPROVED** [review]
Looking good now @kerthcet, thanks for the quick update and the contribution :confetti_ball: 


> [...] i'm glad to join the volunteer work if I met the requirements one day.

And we'd be happy to have you as part of the team :slightly_smiling_face: If you want you can try to review some Pull Requests - just keep the things I pointed out here in mind when you do so :slightly_smiling_face:

**Comment by kerthcet** [comment]
> Looking good now @kerthcet, thanks for the quick update and the contribution 🎊
> 
> > [...] i'm glad to join the volunteer work if I met the requirements one day.
> 
> And we'd be happy to have you as part of the team 🙂 If you want you can try to review some Pull Requests - just keep the things I pointed out here in mind when you do so 🙂

my pleasure. 🚀

**Pull request merged**